woocommerce/woocommerce-android
# Update app illustrations.

This is for #3657 

## To test:

These are the screens where illustrations are updated:

| # | Case 	| Before 	| After: Light 	| After: Dark 	|
|- |-	|-	|-	|-	|
|1. | Empty Orders list: no order. Requires a test site with no order yet. Please note the string change as well. 	|  ![no-order-original](https://user-images.githubusercontent.com/266376/110895139-c52fab00-832b-11eb-9238-f0e00ac1182b.png) | ![no-order-light](https://user-images.githubusercontent.com/266376/110618911-c6e15d80-81c9-11eb-869d-d16340e0b98b.png) | ![no-order-dark](https://user-images.githubusercontent.com/266376/110618902-c5179a00-81c9-11eb-911c-2ce7fe5e4b0d.png) |
|2. | Empty Orders list: all fulfilled. Make sure the site has all orders with the "Completed" status. Probably easiest to create a new site, make one order, then set its status as "Completed". | ![all-order-fulfilled-original](https://user-images.githubusercontent.com/266376/110894778-0f645c80-832b-11eb-88ff-bdb1263acb3a.png)	|  ![all-order-fulfilled-light](https://user-images.githubusercontent.com/266376/110889599-0e2e3200-8321-11eb-9e05-6d37f19a04d3.png)	|  ![all-order-fulfilled-dark](https://user-images.githubusercontent.com/266376/110889605-0ff7f580-8321-11eb-9241-7ccff21f64b1.png)	|
|3. | Order loading. This shows up when opening the "Orders" tab and the app is loading new Orders. | ![order-loading-original](https://user-images.githubusercontent.com/266376/110895012-813ca600-832b-11eb-92de-7b7fc817b484.png) |  ![order-loading-light](https://user-images.githubusercontent.com/266376/110619272-2fc8d580-81ca-11eb-9640-487a06515876.png)	|  ![order-loading-dark](https://user-images.githubusercontent.com/266376/110619268-2dff1200-81ca-11eb-9e11-5e8a6bfb54a0.png) 	|
|4. | Empty search result. Use the search icon on top right of Orders screen, and enter a keyword that will have no result. |  ![no-result-original](https://user-images.githubusercontent.com/266376/110894773-0d9a9900-832b-11eb-912d-92b35578d7aa.png) | ![no-result-light](https://user-images.githubusercontent.com/266376/110619776-c9908280-81ca-11eb-9747-4802ad1eb491.png) 	|  ![no-result-dark](https://user-images.githubusercontent.com/266376/110619772-c85f5580-81ca-11eb-8ad9-8295e1b5657f.png)  	|
|5. | Empty reviews 	| ![empty-reviews-original](https://user-images.githubusercontent.com/266376/110894780-10958980-832b-11eb-97c9-4ca67cdcba96.png)  	|   ![empty-reviews-light](https://user-images.githubusercontent.com/266376/110619552-88986e00-81ca-11eb-9708-6ee526c47c41.png) |  ![empty-reviews-dark](https://user-images.githubusercontent.com/266376/110619554-89c99b00-81ca-11eb-8c00-4b377a546c8f.png)	|
|6. | Empty products 	|  ![no-products-original](https://user-images.githubusercontent.com/266376/110894781-112e2000-832b-11eb-94b7-9437998ed590.png) |  ![no-products-light](https://user-images.githubusercontent.com/266376/110620543-b4682380-81cb-11eb-9303-adb9a3b3bbcc.png) 	| ![no-products-dark](https://user-images.githubusercontent.com/266376/110620547-b5995080-81cb-11eb-9c4e-232dbcaff3f7.png)  	|
|7. | Jetpack required. For this, try logging in to a site with no Jetpack plugin. 	|  ![no-jetpack-original](https://user-images.githubusercontent.com/266376/110892728-2a34d200-8327-11eb-8664-608d50ef06d2.png) |  ![no-jetpack-light](https://user-images.githubusercontent.com/266376/110892381-88ad8080-8326-11eb-95bb-71e10d50a49d.png)	| ![no-jetpack-dark](https://user-images.githubusercontent.com/266376/110892383-89dead80-8326-11eb-9ba9-cb1d69cdfade.png) |

## To test Part 2:

These are the additional updates [as mentioned here](https://github.com/woocommerce/woocommerce-android/pull/3678#issuecomment-814803444) that still needs testing:

| # | Case 	| Before 	| After: Light 	| After: Dark 	|
|- |-	|-	|-	|-	|
|1. | Empty dashboard view	|![image](https://user-images.githubusercontent.com/266376/114407958-266cc780-9bd3-11eb-95ec-26e9bff6c10a.png)|![image](https://user-images.githubusercontent.com/266376/114404928-5d8da980-9bd0-11eb-9979-cbdf90de554e.png)|![image](https://user-images.githubusercontent.com/266376/114405074-80b85900-9bd0-11eb-9812-534de852ecb8.png)|
|2. | Missing analytics view	|![image](https://user-images.githubusercontent.com/266376/114409214-697b6a80-9bd4-11eb-8c40-b71eddeb347e.png)|![image](https://user-images.githubusercontent.com/266376/114409447-ab0c1580-9bd4-11eb-9aad-45db72ebc932.png)	|![image](https://user-images.githubusercontent.com/266376/114409620-d42ca600-9bd4-11eb-8b9d-8755898200f8.png)|
|3. | "What is Jetpack?" login fragment	|![image](https://user-images.githubusercontent.com/266376/114411490-6c775a80-9bd6-11eb-9a8d-ca39d8965958.png)!| ![image](https://user-images.githubusercontent.com/266376/115199156-18f89580-a11d-11eb-9e51-e4ae7e8fae77.png) | ![image](https://user-images.githubusercontent.com/266376/115198475-4a249600-a11c-11eb-9f7a-056cbdeb9c0a.png)|
|4. | "Sign in - Need WordPress" login fragment	|![image](https://user-images.githubusercontent.com/266376/114411796-b52f1380-9bd6-11eb-99a5-a689ef1a1ad3.png)	|![image](https://user-images.githubusercontent.com/266376/114413183-f3790280-9bd7-11eb-8ce2-2efba89e4221.png) | ![image](https://user-images.githubusercontent.com/266376/114413253-05f33c00-9bd8-11eb-8585-9450fb9d7d12.png)|
|5. | "Print Shipping label Info" view	|![image](https://user-images.githubusercontent.com/266376/114848752-3b32a080-9e09-11eb-8abf-d21811077ed8.png) |![image](https://user-images.githubusercontent.com/266376/114849506-05da8280-9e0a-11eb-9632-1e6f23a3fac8.png) |![image](https://user-images.githubusercontent.com/266376/114849585-1a1e7f80-9e0a-11eb-9bba-33d6cd383771.png)	|

## Known issues:

Some illustrations will generate these lint warnings:

> Limit vector icons sizes to 200×200 to keep icon drawing fast; see https://developer.android.com/studio/write/vector-asset-studio#when for more

and

> Very long vector path (xxxx characters), which is bad for performance. Considering reducing precision, removing minor details or rasterizing vector.

For now we won't optimize the illustrations yet, and will handle it in the future in case performance issues do arise. See also p91TBi-4Gb-p2

## Various items that are moved to separate issues:

These items will be resolved outside of this current PR.
- New illustrations that haven't been implemented yet, due to various reasons. They are listed in #3681
- Empty product screen's button needs updating. #3712
- Various design aspects in the app need to be updated to match the design document. #3862

Update release notes:

---
